test/specs/box.spec.js

```diff
@@ -1,7 +1,25 @@
 describe('Box annotation', function() {
   describe('auto', jasmine.fixtures('box'));
 
-  window.testEvents({
+  // event point callbacks
+  const top = function(xScale, yScale, element, xAdjust, yAdjust) {
+    const opts = element.options;
+    return {x: xScale.getPixelForValue(opts.xMin) + xAdjust, y: yScale.getPixelForValue(opts.yMax) + yAdjust};
```

Comment:
Why not use the element properties instead?

Reply:
I wanted to use the "methods" of chartjs to get the coordinates but I think we can use the element properties